src/transpose.c

```diff
@@ -0,0 +1,45 @@
+/* This file contains code to compute the transpose of a given matrix */
+
+#include<stdio.h>
+#include ".\gsl\gsl_matrix.h"
+
+int main()
+{
```

Comment:
According to the coding conventions, starting curly bracket should be at the end of the line and not on a new line.

Reply:
Fixed it. Thanks!!